core/src/test/java/org/web3j/ens/EnsResolverTest.java

```diff
@@ -51,7 +51,7 @@ public void testResolve() throws Exception {
         configureLatestBlock(System.currentTimeMillis() / 1000);  // block timestamp is in seconds
 
         NetVersion netVersion = new NetVersion();
-        netVersion.setResult(Byte.toString(ChainId.MAINNET));
+        netVersion.setResult(Long.toString(ChainId.MAINNET));
```

Comment:
Can we make this backwards compatible? It would require make a new class `ChainIdLong` and deprecating the old one